pm2-explorer.json

```diff
@@ -0,0 +1,15 @@
+{
+        "apps": [{
+                "name": "explorer.app",
+                "script": "app.js",
+                "cwd": ".",
+                "pid_file": "./pids/explorer.app.pid",
+                "out_file": "./logs/explorer.app.log",
```

Comment:
this is a seperate module, not something we implement in the config file. I will add a step to the readme to indicate that it should be installed.